v1/linkml-schemas/document.yaml

```
@@ -102,8 +102,20 @@ classes:
       </ul>
     is_a: DocumentDetailsClass
     slots:
-      - personalNumber   
-
+      - personalNumber 
+  IdCardDetailsClass:
+    description: The details of a National Identification Card as presented by a user for identity verification. 
+      <p>JSON schema&#58; [di_vocab:IdCard](../json-schemas/IdCard.json)</p> 
+      <p>Examples</p>
+      <ul>
+      <li>
+      [Id Card](../examples/IdCardDetailsClass/idCard.json)
```

When written in English, I'd write "ID", despite the strange things we end up having to do with the capitalisation elsewhere :)

```suggestion
      [ID Card](../examples/IdCardDetailsClass/idCard.json)
```